mods/ca/rules/player.yaml

```diff
@@ -199,21 +199,21 @@ Player:
 	LobbyPrerequisiteCheckbox@BALANCEDHARVESTING:
 		ID: balancedharvesting
 		Label: Balanced Harvesting
-		Description: Enables dynamic harvester speed to account for the direction of resources relative to refineries
+		Description: Enables dynamic harvester speed to account for the direction of resources relative to refineries.
 		Enabled: True
 		DisplayOrder: 15
 		Prerequisites: global.balancedharvesting
 	LobbyPrerequisiteCheckbox@NukularConyard:
 		ID: nukular
-		Label: Nukular MCV
-		Description: Enables Nukular MCV Mode - The first conyard of each player is nukular,\nthis means if you lose it, you lose the game and it will explode in a gigantic nuclear detonation. \nGreat for FFA.
+		Label: Nuclear MCV
```

Comment:
Nukular was supposed to be a reference to homer simpson saying nuclear in a strange way telling others how it should be correctly said, so it was on purpose not correct orthography, but if you think there is something better than Nukular, make a proposal